### ISSUE TITLE
SwiftUI Error - Publishing changes from within view updates is not allowed, this will cause undefined behavior.

### ISSUE THREAD

**Issue by devinhayward** [mergesort/Boutique#31]
I am trying to import data through a CSV file. Using .fileImporter (SwiftUI) and within the closure parsing the data and attempting to save to the Store.

The compiler then complains:

/SourcePackages/checkouts/Boutique/Sources/Boutique/Stored.swift:42 Publishing changes from within view updates is not allowed, this will cause undefined behavior.

Not sure if I am doing some undefined behaviour for the Boutique Package, or using it in the wrong way?

Thought I would point out this bug to your attention. Maybe something you are not aware of.

**Comment by mergesort** [comment]
- Hey @devinhayward, thanks for the bug report! Unfortunately I've looked into the issue when it started during the beta period but it appears that it's an issue with Xcode 14. I don’t have a fix for it, and I found a blog post that led to some conversations with the author of [this blog post](https://www.donnywals.com/xcode-14-publishing-changes-from-within-view-updates-is-not-allowed-this-will-cause-undefined-behavior/), but it seems the behavior is undefined or a bug in the latest SDK. The good news is that I've been using the for a month or two and haven't run into any problems, and I'm sad to say that the real fix likely lies in the Apple fixing the error or explaining how to properly handle this situation.
